contracts/its-stacks-translator/src/contract/its_hub_message_to_clarity_bytes.rs

```diff
@@ -0,0 +1,342 @@
+        ),
+        (
+            ClarityName::from(CLARITY_NAME_AMOUNT),
+            Value::UInt(amount.u128()),
+        ),
+        (
+            ClarityName::from(CLARITY_NAME_DATA),
+            Value::buff_from(if let Some(data) = data {
+                data.to_vec()
+            } else {
+                vec![]
+            })
+            .map_err(|_| ContractError::InvalidPayload)?,
+        ),
+    ])?;
+
+    Ok(Value::from(tuple_data).serialize_to_vec()?)
+}
+
+fn get_its_deploy_interchain_token_payload_receive_from_hub(
+    source_chain: ChainNameRaw,
+    token_id: TokenId,
+    name: String,
+    symbol: String,
+    decimals: u8,
+    minter: Option<nonempty::HexBinary>,
+) -> Result<Vec<u8>, ContractError> {
+    let token_id: [u8; 32] = token_id.into();
+
+    let tuple_data = TupleData::from_data(vec![
+        (
+            ClarityName::from(CLARITY_NAME_TYPE),
+            Value::UInt(MESSAGE_TYPE_DEPLOY_INTERCHAIN_TOKEN),
+        ),
+        (
+            ClarityName::from(CLARITY_NAME_SOURCE_CHAIN),
+            Value::string_ascii_from_bytes(source_chain.to_string().into_bytes())
+                .map_err(|_| ContractError::InvalidPayload)?,
+        ),
+        (
+            ClarityName::from(CLARITY_NAME_TOKEN_ID),
+            Value::buff_from(token_id.to_vec()).map_err(|_| ContractError::InvalidPayload)?,
+        ),
+        (
+            ClarityName::from(CLARITY_NAME_NAME),
+            Value::string_ascii_from_bytes(name.into_bytes())
+                .map_err(|_| ContractError::InvalidPayload)?,
+        ),
+        (
+            ClarityName::from(CLARITY_NAME_SYMBOL),
+            Value::string_ascii_from_bytes(symbol.into_bytes())
+                .map_err(|_| ContractError::InvalidPayload)?,
+        ),
+        (
+            ClarityName::from(CLARITY_NAME_DECIMALS),
+            Value::UInt(decimals.into()),
+        ),
+        (
+            ClarityName::from(CLARITY_NAME_MINTER_BYTES),
+            Value::buff_from(if let Some(minter) = minter {
+                minter.to_vec()
+            } else {
+                vec![]
+            })
+            .map_err(|_| ContractError::InvalidPayload)?,
+        ),
+    ])?;
+
+    Ok(Value::from(tuple_data).serialize_to_vec()?)
+}
+
+#[cfg(test)]
+mod tests {
+    use std::str::FromStr;
+
+    use axelar_wasm_std::nonempty;
+    use clarity::vm::representations::ClarityName;
+    use clarity::vm::types::{TupleData, Value};
+    use cosmwasm_std::HexBinary;
+    use interchain_token_service_std::{
+        DeployInterchainToken, HubMessage, InterchainTransfer, Message, TokenId,
+    };
+    use router_api::ChainNameRaw;
+
+    use crate::contract::its_hub_message_to_clarity_bytes::{
+        its_hub_message_to_clarity_bytes, MESSAGE_TYPE_DEPLOY_INTERCHAIN_TOKEN,
+        MESSAGE_TYPE_INTERCHAIN_TRANSFER,
+    };
+    use crate::error::ContractError;
+
+    #[test]
+    fn test_its_hub_message_to_clarity_bytes_error() {
```

Comment:
```suggestion
    fn its_hub_message_to_clarity_bytes_error() {
```
test prefixes are unnecessary since it's implied by the label `#[test]`

Reply:
Done